### PR TITLE
[ASM] RankServiceOwners Bug Fix - EXPANDR-10828

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-10828

## Description
Fixed issues where RankServiceOwners wasn't always running, wasn't working for single service owner entry, and in the case of not populating service owners was displaying a misleading war room entry

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , please review

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_49.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+##### Cortex ASM - Service Ownership
+
+- Fixed an issue where the `RankServiceOwners` script could get skipped.
+
+#### Scripts
+
+##### RankServiceOwners
+
+- Fixed an issue where single entry service owner list wasn't getting parsed correctly.
+- Updated the war room message to reflect if the `asmserviceowner` field was updated.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_49.md:13]:
```suggestion
- Updated the War Room message to reflect if the `asmserviceowner` field was updated.
```

**Comment by ShirleyDenkberg** [comment]
@israelpoli @tomer-pan Doc review completed.

File: Packs/CortexAttackSurfaceManagement/Scripts/RankServiceOwners/RankServiceOwners_test.py [hunk 2]
```diff
@@ -478,7 +489,10 @@ def test_main(mocker, owners, asm_system_ids, expected_out, capfd):
 
     # Verify the output value was set
     expected_calls_to_mock_object = [unittest.mock.call('setAlert', {'asmserviceowner': expected_out})]
-    assert demisto_execution_mock.call_args_list == expected_calls_to_mock_object
+    if demisto_execution_mock.call_args_list:
+        assert demisto_execution_mock.call_args_list == expected_calls_to_mock_object
+    else:
+        assert demisto_execution_mock.call_args_list == []
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Scripts/RankServiceOwners/RankServiceOwners_test.py:495]: Should we expect an empty list if no owners are passed in?

File: Packs/CortexAttackSurfaceManagement/pack_metadata.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "name": "Cortex Attack Surface Management",
     "description": "Content for working with Attack Surface Management (ASM).",
     "support": "xsoar",
-    "currentVersion": "1.7.48",
+    "currentVersion": "1.7.49",
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/pack_metadata.json:5]:
```suggestion
    "currentVersion": "1.7.50",
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_49.md
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_49.md:1]: change to [1_7_50.md](https://github.com/demisto/content/pull/36250/files#diff-49b9c2576407082f9ad6348d69030f8a02c241767cade8875df0b89f20d151b4)

**Review by BigEasyJ — APPROVED** [review]

File: Packs/CortexAttackSurfaceManagement/Scripts/RankServiceOwners/RankServiceOwners_test.py (hunk 2 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Scripts/RankServiceOwners/RankServiceOwners_test.py:495]: yes, would only happen if we only have service-account owners

**Comment by johnnywilkes** [comment]
@israelpoli  / @tomer-pan , this has been approved by Xpanse.  Please merge when you can, please

**Review by israelpoli — APPROVED** [review]
Hi @johnnywilkes
I merge it and I fix the RN version in internal PR

🙏

**Pull request merged**